src/app/modals/BasicAboutModal.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { AboutModal, Button, TextContent, TextList, TextListItem } from '@patternfly/react-core';
+import brandImg from './pf_mini_logo_white.svg';
+import logoImg from './pf_logo.svg';
+
+class BasicAboutModal extends React.Component <{}, {isModalOpen: boolean}>{
```

Comment:
Please, define types separately. Like here for example https://github.com/fusor/mig-ui/blob/736d36663cd99bcc96b162050b3dbd9da3d8ec58/src/app/AppComponent.tsx#L9

Reply:
Added